### PR TITLE
Feat/create resources page

### PR DESCRIPTION
**Pull request by IsaacDMz**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Related issue

- #561 

image

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks @IsaacDMz! I'll get to this soon :)

File: frontend/i18n/en-US.json [hunk 1]
```diff
@@ -370,6 +370,19 @@
...
   "pages.events.team.tagline": "Event organizers",
+  "pages.resources.create.header": "Information",
+  "pages.resources.create.subtext": "Thanks for linking a resource to activist! This is how you can keep all the tools and information that you need to get active in one place. Please only link resources that you have the right to share with others. You can learn more about our terms and confitions for copyrighted materials on the resources page of the documentation",
+  "pages.resources.create.resource-name": "Name",
+  "pages.resources.create.resource-name-placeholder": "A name for the resource",
+  "pages.resources.create.link": "Link",
+  "pages.resources.create.link-placeholder": "A URL to the resource",
+  "pages.resources.create.description": "Short description",
+  "pages.resources.create.description-placeholder": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
+  "pages.resources.create.location":"Location",
```
Comment (andrewtavis) [inline frontend/i18n/en-US.json:381]: Note that this should be combined with the other "Location" i18n key. Ping @to-sta that an error for the workflow wasn't triggered :)

Comment (IsaacDMz) [reply frontend/i18n/en-US.json:381]: Should i use the organization location on this key, or add Location to global keys?

Comment (andrewtavis) [reply frontend/i18n/en-US.json:381]: You can check the Python script that was supposed to fail on the branch and should fail locally :) Run `python frontend/i18n/check_repeat_i18n_values.py` and it should list this values that have been repeated and also suggest what the lowest level of key should be.

Comment (andrewtavis) [reply frontend/i18n/en-US.json:381]: Looks like we can do `"pages._global.location": "Location"` :) Give it a check though 😊

Comment (andrewtavis) [reply frontend/i18n/en-US.json]: Nice @IsaacDMz :) Thanks! I'll get to this in the coming days. My home internet's a bit screwed up atm, and I don't like reviewing on my phone's hotspot 😅

**Review by andrewtavis — APPROVED** [review]
Very solid work, @IsaacDMz 😊 Thanks for coming in quick and fixing the i18n "issues" even though the workflow failed. Really appreciate all of this!

As we have the `/resources` being linked to via the breadcrumbs I'll go ahead and add a placeholder page in there. We can even do some quick page analytics and just do a list of the resources in order of how much they've been viewed for the time being 😊 Will close the issue when that's all done!

**Pull request merged**